phone/src/os/notifications/components/NotificationBar.tsx

```diff
@@ -76,6 +76,7 @@ export const NotificationBar = () => {
         className={classes.root}
         container
         justifyContent="space-between"
+        alignItems="center"
```

Comment:
This will only center the 'time' horizontally. I thought the issue here was that the vertical alignment between the camera on the frame and the 'time' was not in center? @RockySouthpaw Please confirm.